panic while execute accept command

Base on the README.md file, while execute 'accept' command it panic:


/tmp/appraise:  git appraise accept 41592
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xa0 pc=0x121de63]

goroutine 1 [running]:
github.com/google/git-appraise/review.GetSummary(0x13d8f20, 0xc4200771f0, 0x7fff5fbff1f4, 0x5, 0xc42004fd00, 0x0, 0x0)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/review/review.go:245 +0x173
github.com/google/git-appraise/review.Get(0x13d8f20, 0xc4200771f0, 0x7fff5fbff1f4, 0x5, 0x0, 0x0, 0x0)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/review/review.go:286 +0x4d
github.com/google/git-appraise/commands.acceptReview(0x13d8f20, 0xc4200771f0, 0xc42007c0b0, 0x1, 0x1, 0x100d374, 0x12b7601)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/commands/accept.go:48 +0xb7
github.com/google/git-appraise/commands.glob..func4(0x13d8f20, 0xc4200771f0, 0xc42007c0b0, 0x1, 0x1, 0xc420064d00, 0x9)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/commands/accept.go:93 +0x53
github.com/google/git-appraise/commands.(*Command).Run(0x13f60c0, 0x13d8f20, 0xc4200771f0, 0xc42007c0b0, 0x1, 0x1, 0x0, 0xc42007cdb0)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/commands/commands.go:39 +0x5c
main.main()
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/git-appraise/git-appraise.go:100 +0x38c

/tmp/appraise(✔) git appraise accept f38f
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xa0 pc=0x121de63]

goroutine 1 [running]:
github.com/google/git-appraise/review.GetSummary(0x13d8f20, 0xc4200771f0, 0x7fff5fbff1f4, 0x4, 0xc42004fd00, 0x0, 0x0)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/review/review.go:245 +0x173
github.com/google/git-appraise/review.Get(0x13d8f20, 0xc4200771f0, 0x7fff5fbff1f4, 0x4, 0x0, 0x0, 0x0)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/review/review.go:286 +0x4d
github.com/google/git-appraise/commands.acceptReview(0x13d8f20, 0xc4200771f0, 0xc42007c0b0, 0x1, 0x1, 0x100d374, 0x12b7601)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/commands/accept.go:48 +0xb7
github.com/google/git-appraise/commands.glob..func4(0x13d8f20, 0xc4200771f0, 0xc42007c0b0, 0x1, 0x1, 0xc420064d00, 0x9)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/commands/accept.go:93 +0x53
github.com/google/git-appraise/commands.(*Command).Run(0x13f60c0, 0x13d8f20, 0xc4200771f0, 0xc42007c0b0, 0x1, 0x1, 0x0, 0xc42007cdb0)
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/commands/commands.go:39 +0x5c
main.main()
	/Users/jiaxi/code/golang/src/github.com/google/git-appraise/git-appraise/git-appraise.go:100 +0x38c
/tmp/appraise(✔) git appraise accept f83fa03814f0

the work-around is effectively.

--- a/review/review.go
+++ b/review/review.go
@@ -217,7 +217,7 @@ func (r *Summary) loadComments(commentNotes []repository.Note) []CommentThread {
 func getSummaryFromNotes(repo repository.Repo, revision string, requestNotes, commentNotes []repository.Note) (*Summary, error) {
 	requests := request.ParseAllValid(requestNotes)
 	if requests == nil {
-		return nil, nil
+		return nil, fmt.Errorf("Could not find any review requests for %q", revision)
 	}
 	sort.Stable(requestsByTimestamp(requests))
 	reviewSummary := Summary{
@@ -235,6 +235,9 @@ func getSummaryFromNotes(repo repository.Repo, revision string, requestNotes, co
 //
 // If no review request exists, the returned review summary is nil.
 func GetSummary(repo repository.Repo, revision string) (*Summary, error) {
+	if err := repo.VerifyCommit(revision); err != nil {
+		return nil, fmt.Errorf("Could not find a commit named %q", revision)
+	}
 	requestNotes := repo.GetNotes(request.Ref, revision)
 	commentNotes := repo.GetNotes(comment.Ref, revision)
 	summary, err := getSummaryFromNotes(repo, revision, requestNotes, commentNotes)

